Source/missiles.cpp

```diff
@@ -701,7 +701,8 @@ bool GrowWall(int playerId, Point position, Point target, MissileID type, int sp
 		return false;
 	}
 
-	AddMissile(position, position, Players[playerId]._pdir, type, TARGET_BOTH, playerId, damage, spellLevel);
+	const Player &player = Players[playerId];
+	AddMissile(position, position, player._pdir, type, TARGET_BOTH, player, damage, spellLevel);

```

Comment:
I get why, but it feels a bit odd to look up the player by id just to look up the id based on offset in the next line once the function is inlined :/

Reply:
I have changed the logic in `GrowWall` to not use a player at all and to always use south direction instead.
Reason: The player was only used to get the direction (`_pdir`). But the direction is ignored in `AddMissile` because the firewall sprites only have only one direction (frame < 8).